lib/ulid/generate.rb

```diff
@@ -35,6 +35,10 @@ def encode10
       (hi << 64) | lo
     end
 
+    def encode_hex
```

Comment:
I wonder if this should be called `encode16` and `encode_uuid` the exception? Alternatively, I wonder if `encode16` should have a param such as `encode16(uuid_delimiter="-")`  which you can set to `""`?

Reply:
in that case `encode_uuid` and `decode_uuid` should be the exceptions and `base16` the default